jdaviz/configs/default/plugins/gaussian_smooth/tests/test_gaussian_smooth.py

```diff
@@ -93,8 +93,8 @@ def test_linking_after_spectral_smooth(cubeviz_helper, spectrum1d_cube):
 
     label_mouseover._viewer_mouse_event(spec_viewer,
                                         {'event': 'mousemove', 'domain': {'x': 4.6236e-7, 'y': 60}})
-    assert label_mouseover.as_text() == ('', '', '')
-    assert label_mouseover.icon == ''
+    assert label_mouseover.as_text() == ('Cursor 4.62360e-07, 6.00000e+01', '', '')
+    assert label_mouseover.icon == 'mdi-cursor-default'
```

Comment:
instead of showing nothing, we can just show cursor position (and with the cursor icon instead of no icon)